app/src/main/java/org/schabi/newpipe/local/playlist/LocalPlaylistFragment.java

```diff
@@ -472,13 +472,57 @@ public void removeWatchedStreams(final boolean removePartiallyWatched) {
                         "Removing watched videos, partially watched=" + removePartiallyWatched))));
     }
 
+    /**
+     * If the current thumbnail has changed but the video is still the same, this method will update
+     * the playlist thumbnail accordingly.
+     *
+     * @param currentItems List of items currently in the playlist
+     * @param updatedItems List of items that will replace the current items in the playlist
+     */
+    private void updateCurrentThumbnail(final List<LocalItem> currentItems,
+                                        final List<PlaylistStreamEntry> updatedItems) {
+
+        final String playlistThumbnail = playlistManager.getPlaylistThumbnail(playlistId);
+        long streamId = -1;
```

Comment:
Could you please avoid magic numbers?

Reply:
I added a comment explaining this variable/number. I hope this is understandable now.